src/Symfony/Component/HttpKernel/CacheClearer/Psr6CacheClearer.php

```diff
@@ -59,6 +57,10 @@ public function clearPool(string $name): bool
 
     public function clear(string $cacheDir): void
     {
+        if (!isset($this->pools)) {
+            return;
+        }
```

Comment:
If downstream code overrides this class without calling the parent constructor, `$this->pools` can be uninitialized at this point. This wasn't the case before my change.

Reply:
`Psr6CacheClearer` doesn't provide anything if `$pools` is not initialized. Why would you support it?

Reply:
> Why would you support it?

If I wrote this class from scratch, I wouldn't. I merely being extra careful because I've remove a property initialization. That being said, I'm not attached to that piece of code. I can remove it.

Reply:
Let's revert CPP on this class instead. Since we want the property to be initialized.

Reply:
Done.